astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -12,6 +12,7 @@
 from numpy.testing import assert_allclose, assert_array_equal
 
 from astropy import units as u
+from astropy.coordinates import Angle
```

Comment:
I'd prefer not to import from elsewhere in `astropy` (hoping to at some point move `units` to its own package...). See below.